pages/_app/index.module.css

```diff
@@ -1,9 +1,3 @@
-html,
-body,
-:global(#__next) {
-  height: 100%;
-}
-
```

Comment:
Where did this go? I cannot find it anywhere?

Reply:
https://github.com/oacore/dashboard/blob/upgrade-next/pages/_app/global.css